src/nvim/main.c

```diff
@@ -145,6 +145,21 @@ void early_init(void)
 {
   handle_init();
 
+  // Use $NVIM and $NVIMRUNTIME as $VIM and $VIMRUNTIME
+  // or unset them to avoid accidentaly using the Vim runtime
+  const char *env_nvimruntime = os_getenv("NVIMRUNTIME");
+  if (env_nvimruntime) {
+    os_setenv("VIMRUNTIME", env_nvimruntime, true);
+  } else {
+    os_unsetenv("VIMRUNTIME");
+  }
+  const char *env_nvim = os_getenv("NVIM");
+  if (env_nvim) {
+    os_setenv("VIM", env_nvim, true);
```

Comment:
Wouldn't that break scripts all over? i.e. if VIM is not set and we don't set it in `vim_getenv()` $VIM will never hold the value expected by scripts.